FsAst.fsproj

```diff
@@ -21,6 +21,7 @@
     <Compile Include="CreateAst.fs" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.3.4" />
```

Comment:
that's not enough, you need also to disable the implicit packagereference

```
<DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
<DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
```

Reply:
I'm confused why you would need `DisableImplicitFSharpCoreReference`.

Reply:
Happy to merge a pull request for that. Thanks for the other pull requests! Do you want a new version published soon?